### DISCUSSION TITLE
Why VSCode is opened?

### DISCUSSION THREAD

**Discussion by linonetwo** [RosettaCommons/RFdiffusion#226] [Q&A] [answered]
When I run `./scripts/run_inference.py 'contigmap.contigs=[150-150]' inference.output_prefix=test_outputs/test inference.num_designs=10`

It open `run_inference.py` in vscode, and VSCode will take up to 8GB memory, why?

**Comment by roccomoretti** [comment, marked as answer]
This is almost certainly due to the settings on your computer. When you installed VSCode, you likely instructed it to open Python files, and it changed the settings such that it takes priority in opening those files, rather than having the python scripts be executed with the Conda-installed python which normally should be used to run RFdiffusion scripts.

You'll have to investigate the VSCode settings on you machine to tell it not to open Python file when you put them on the command line to run. (Consult VSCode documentation and help forums for details.)

You also want to make sure your Conda is installed properly and properly activated such that the Conda environment you created when installing RFdiffusion will be used when you launched the run_inference.py script.

**Reply by linonetwo** [reply]
Thank you, this explains it.

Short answer: use `python ./scripts/run_inference.py` (prepend it with `python`)

i.g. `python ./scripts/run_inference.py 'contigmap.contigs=[150-150]' inference.output_prefix=test_outputs/test inference.num_designs=10` for the example in readme.
